src/TaskRunner/Commands/ComponentCheckCommands.php

```diff
@@ -87,21 +92,22 @@ public function componentCheck(ConsoleIO $io, array $options = [
 
         // Execute all checks.
         $checks = [
-            'Mandatory',
-            'Recommended',
-            'Insecure',
-            'Outdated',
-            'Abandoned',
-            'Unsupported',
-            'Evaluation',
-            'Development',
-            'Composer',
-            'Configuration',
+            'componentMandatory' => 'Mandatory',
+            'componentRecommended' => 'Recommended',
+            'componentInsecure' => 'Insecure',
+            'componentOutdated' => 'Outdated',
+            'componentAbandoned' => 'Abandoned',
+            'componentUnsupported' => 'Unsupported',
+            'componentEvaluation' => 'Evaluation',
+            'componentDevelopment' => 'Development',
+            'componentComposer' => 'Composer',
+            'componentConfiguration' => 'Configuration',
+            'componentNpmInsecure' => 'Npm Insecure',
```

Comment:
Make these checks conditional, only are processed if in .opts.yml the npm_install is true,